server/runtime/src/main/server/server/conf/infinispan-dev-mode.xml

```diff
@@ -35,10 +35,15 @@
    </jgroups>
 
    <cache-container name="default" statistics="true">
+      <metrics accurate-size="true"/>
       <transport cluster="${infinispan.cluster.name:cluster}" stack="${infinispan.cluster.stack:tcp}" node-name="${infinispan.node.name:}"/>
       <security>
          <authorization/>
       </security>
+      <local-cache name="default"/>
```

Comment:
Replace with a distributed cache that inherits from the template defined below.

Reply:
I pushed a change